spec/features/quotes/index_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+RSpec.describe 'Returning Quotes' do
+  describe "happy path" do
+    before(:each) do
+      visit root_path
+      fill_in :keyword, with: 'I am'
+      click_on "Search for Quotes"
+    end
+    it "displays the total number of quotes" do
+      quote_count = QuotesFacade.searched_quotes("I am").count
+      expect(page).to have_content("Total Matching Quotes: #{quote_count}")
+    end
+
+    it "displays a list of the top ten quotes" do
+
+    end
+
+    it "has information on each quote" do
+
+    end
+  end
+
+  describe "sad paths" do
+
+    xit "if there are no tags associated with a quote, it does not display the tag category" do
```

Comment:
Reach out to some of your peers to see how they thought about testing for no tags.